docs/Q-and-A.md

```diff
@@ -0,0 +1,635 @@
+      - [Public Key Infrastructure](./Glossary.md#public-key-infrastructure)
+      - [Race condition](./Glossary.md#race-condition)
+      - [Root of trust](./Glossary.md#root-of-trust)
+      - [Secret](./Glossary.md#secret)
+      - [Self Sovereign Identity](./Glossary.md#self-sovereign-identity)
+      - [Subject](./Glossary.md#subject)
+      - [Transfer](./Glossary.md#transfer)
+      - [Transferable identifier](./Glossary.md#transferable-identifier)
+      - [Trust-over-IP](./Glossary.md#trust-over-ip)
+      - [Validator](./Glossary.md#validator)
+      - [Verifiable Credential](./Glossary.md#verifiable-credential)
+      - [W3C DID](./Glossary.md#w3c-did)
+      - [WebAssembly](./Glossary.md#webassembly)
+      - [(Digital Identity) Wallet](./Glossary.md#-digital-identity--wallet)
+
+## Knowledge you should be confidently applying
+- The definitions above
+- W3C DIDs
+- Public private key pairs
+- Bitcoin Improvement Protocols: BIP32, BIP39, BIP44, BIP47, BIP49, BIP84, BIP174
+- hierarchical deterministic derivation paths
+- Base58
+- IPFS
+- TBW {prio 2}
+## Actions you should be comfortable with
+- Amend knowledge and keep existing knowledge up to date
+- create a key pair safely and back it up safely
+- TBW {prio 2}
+
+# Jump table to categories
+- [General](#qa-section-general)
+- [Sidetree operational](#qa-section-sidetree-operational)
+- [Userinterface](#qa-section-userinterface)
+- [Identifiers](#qa-section-identifiers)
+- [Wallets](#qa-section-wallets)
+- [Signatures](#qa-section-signatures)
+- [Private Key Management](#qa-section-private-key-management)
+- [Blockchain](#qa-section-blockchain)
+- [Agencies](#qa-section-key-agencies)
+- [ION](#qa-section-ION)
+
+# Q&A section General
+
+## *Q: What is Sidetree?
+Sidetree is a protocol for creating a scalable decentralized public key infrastructure ([DPKI](./Glossary.md#public-key-infrastructure)) networks that can run atop of any existing decentralized ledger system (e.g. Bitcoin) and be as open, public, and permissionless as the underlying ledger they utilize.
+
+#### **Q: Why only a protocol, why scalable, why decentralized, why a ledger, why permissionless, why open and public?
+1. Sidetree is a _protocol_ because we need it to be multi applicable. Implementers will be provided with software tools and toolkits and the protocol has both `normative` and non-normative guidelines.
+2. The _scalability_ feature is needed because of the astronomical numbers that are associated with digital identifiers, the numerous life events of each identifier and the mulitple use of each individual identifier. Every individual, group or thing on earth can have multiple identifiers associated with them.
+3. Privacy and guarding control over are the main drivers for the deer wish to _decentralize_ identifiers. If you don’t like being tracked and you don’t like anybody else controlling your identity, decentralization is the way to go. 
+4. A global _ledger_ is needed as a single source of truth.
```

Comment:
A single global ledger is not needed for a Sidetree implementation. Some sidetree implementations make this choice (e.g., did:ion) while other Sidetree implementations make more ad-hoc choices (e.g., did:orb).

Reply:
thx, I just your exact wording under point 4